My Series List/My Series List/Model/SeriesModel.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  SeriesModel.swift
+//  My Series List
+//
+//  Created by Ivan Dias on 18/11/2021.
+//
+
+import Foundation
+
+enum Api {
+    
+    struct Results: Codable {
+        let results: [Series]
+    }
+    
+    struct Series: Codable {
+        let id: Int?
+        let name: String?
+        let overview: String?
+        let posterPath: String?
+        let voteAverage: Double?
+        
+        enum CodingKeys: String, CodingKey {
+            case id
```

Comment:
Can you remove id, name and overview from CodingKeys?

Reply:
I got an error when I remove those properties, I searched for info and I found that I have to declare all properties inside CodingKeys